handler.go

```diff
@@ -28,6 +33,10 @@ func validateLambdaHandler(lambdaHandler interface{}) (bool, error) {
 	return takesContext, nil
 }
 
+// validateArguments validates whether the arguments passed as part of the lambdaHandler are valid. A valid lambdaHandler
+// has a maximum of two arguments. When there are two arguments, the first one must be a Context. The function returns
+// true or false depending on whether the lambdaHandler has a context argument. If the arguments are not valid, an error
```

Comment:
Probably add reference link - https://docs.aws.amazon.com/lambda/latest/dg/go-programming-model-handler-types.html for more information.

Reply:
Added the reference